### PR TITLE
Branch dg: Address Book feature

### PR DESCRIPTION
**Pull request by lohszeying**

### CODE REVIEW

**Comment by BryanYap972** [comment]
Looks fine, can merge.

**Review by BryanYap972 — COMMENTED** [review]
Looks fine.

**Comment by lohszeying** [comment]
@xCelestialPhoenix Please check and merge if it looks fine, thanks~!

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -308,11 +308,21 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
```
Comment (xCelestialPhoenix) [inline docs/DeveloperGuide.adoc:310]: Missing the "so that I..." portion.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -308,11 +308,21 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |student |add profile picture to added contact |know at a glance who is the person in my contact
 
-|`* * *` |student |edit profile picture to added contact |edit the picture if changes are necessary
+|`* * *` |student |delete profile picture to added contact |
```
Comment (xCelestialPhoenix) [inline docs/DeveloperGuide.adoc:313]: Missing the "so that I..." portion.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -308,11 +308,21 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* * *` |student | delete category specific remark |category specific remark is not needed anymore
+
+|`* * *` |student |list all my contacts |keep track of my contact list
+
+|`* * *` |student |list my contacts according to category |keep track of my contact list according to different category
+
+|`* * *` |student |undo my command under Address Book feature |
+
+|`* * *` |student |redo my command under Address Book feature |
```
Comment (xCelestialPhoenix) [inline docs/DeveloperGuide.adoc:327]: Missing the "so that I..." portion for both undo and redo.

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -308,11 +308,21 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |student |add profile picture to added contact |know at a glance who is the person in my contact
 
-|`* * *` |student |edit profile picture to added contact |edit the picture if changes are necessary
+|`* * *` |student |delete profile picture to added contact |
 
-|`* * *` |student |delete profile picture to added contact|
+|`* * *` |student |add category specific remark |filter out a contact’s remark according to type of contact
```
Comment (xCelestialPhoenix) [inline docs/DeveloperGuide.adoc:315]: Missing the word "can" before filter. The header doesn't have "can". Same for some of your other user stories.

**Review by xCelestialPhoenix — COMMENTED** [review]
I haven't really standardize the DG so if can, try to format according to my user stories (Starting from the "I can view my schedule for the day"). It will make my life easier. Thanks.

**Pull request closed without merging**